app/src/shared/components/ContextMenu/index.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import React, { type ElementType } from 'react'
+import { Popover } from 'reactstrap'
+
+import ContextMenuItem from './ContextMenuItem'
+import styles from './contextMenu.pcss'
+
+export type Props = {
+    target: ElementType,
+    placement: string,
+    isOpen: boolean,
+    children: Array<ContextMenuItem>,
+}
+
+class ContextMenu extends React.Component<Props> {
+    static Item = ContextMenuItem
+
+    render = () => {
+        const { target, placement, isOpen, children } = this.props
+        return (
+            <Popover
+                isOpen={isOpen}
+                placement={placement}
+                target={target}
+                className={styles.menu}
```

Comment:
Always good to allow components to be passed custom `className` prop, i.e. `className={cx(props.className, styles.menu)}`